tests/test_013_solinst.py

```diff
@@ -0,0 +1,54 @@
+# %%
+import logging
+
+from hydropandas.io import solinst
+from hydropandas import observation
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+# %% test observations
+
+
+def test_read_solinst_file_obs():
+    # observation of slugtest, 8 observations per second
+    df, meta = solinst.read_solinst_file(
+        "./tests/data/2024-solinst-test/WsNoo_dp366_BUB_20231222_slug1m.xle",
+    )
+
+    assert len(df) == 587, "Dataframe should have 587 readings"
+
+def test_read_solinst_file_meta_has_location():
+    # observation of slugtest, created via iOs app, location included by app
+    df, meta = solinst.read_solinst_file(
+        "./tests/data/2024-solinst-test/WsNoo_dp366_BUB_20231222_slug1m.xle",
+    )
+
+    assert meta['x'] == 52730.58, "x coordinate should be 52730.58"
+
+def test_read_solinst_file_meta_without_location():
+    # example observation created via desktop, location not included
+    df, meta = solinst.read_solinst_file(
+        "./tests/data/2024-solinst-test/example-10min-interval-via-laptop.xle",
+    )
+
+    assert meta['x'] is None, "x coordinate not available"
+
+def test_read_solinst_file_with_manual_meta():
+    # manual metadata about levels provided
+    screen_bottom = -10,
+    screen_top = -5,
+    ground_level = -1,
+    tube_nr = 1,
+    tube_top = -0.5
```

Comment:
these were not meant to be tuples i don't think?

Reply:
A trailing comma turns the `screen_top = 5,` into a tuple instead of an integer. I removed the comma's in my changes, so should be fine now.

Reply:
Ah. The commas. Thanks.